libs/login/gradle.properties-example

```diff
@@ -0,0 +1,17 @@
+# Project-wide Gradle settings.
+
+# Optional: used to autofill username and password fields at login on debug build only
```

Comment:
Suggestion (💡): Consider removing everything but these 4 `wp` related properties from this example `gradle.properties` file.

```properties
# Optional: used to autofill username and password fields at login on debug build only
wp.debug.wpcom_login_email =
wp.debug.wpcom_login_username =
wp.debug.wpcom_login_password =
wp.debug.wpcom_website_url =
```

FYI: The rest of the properties are anyway matching the parent `gradle.properties` file (`android.useAndroidX=true`, etc).

PS: Same applies to the example `gradle.properties` file on the `fluxc` level, the recommended change being:

```properties
# Optional: used to enable Wordpress API and database downgrade on debug build only
fluxc.ENABLE_WPAPI = false
wp.ENABLE_DATABASE_DOWNGRADE = false
```

Btw, I am not sure why this `gradle.properties-example` file is not shown on this diff, but it is anyway shown on my branch after me checking-out it via this PR, weird... 🤷

Reply:
This is done as part of #21353 and c85ec844790477331e08ee546a807acb6e6aaeb6.